go-controller/pkg/ovn/egressgw.go

```diff
@@ -159,6 +181,23 @@ func (oc *Controller) deletePodGWRoutesForNamespace(pod, namespace string) {
 			}
 			mask := GetIPFullMask(podIP)
 			// TODO (trozet): use the go bindings here and batch commands
+			if config.Gateway.Mode == config.GatewayModeHybrid {
+				// delete allow policy to bypass lr-policy in GR
+				var l3Prefix string
```

Comment:
Please make this a separate function and call it from all 3 locations.

Reply:
done